plugins/services/src/js/utils/ServiceUtil.js

```diff
@@ -135,27 +108,48 @@ const ServiceUtil = {
     return serviceName[serviceName.length - 1];
   },
 
-  convertServiceLabelsToArray(service) {
-    if (!(service instanceof Service)) {
-      return [];
-    }
-
-    const labels = service.getLabels();
-    if (labels == null) {
-      return [];
-    }
-
-    return Object.keys(labels).map(key => ({ key, value: labels[key] }));
+  isSDKService(service) {
+    return (
```

Comment:
why are we dropping the `if (!(service instanceof Service))` check? 
It feels rather safe to test `getLabels` existence though having the function doesn't mean the output will be expected. Checking on a class would be much a safer bet.

Reply:
it didnt exist in `src/js/ServiceUtil` which was used by `Service` before… if we add it, we're creating circular dependency 😞